### PR TITLE
docs: removed old k8s demo and refer kube-thanos instead, also update…

### PR DESCRIPTION
**Pull request by FUSAKLA**
… release process to update tutorials

As discussed on slack https://cloud-native.slack.com/archives/CL25937SP/p1568848738006400

- removed old `kubernetes-demo`
- added updating of tutorials to release process
- refer kube-thanos in getting started since we removed the last k8s manifest examples.

@bwplotka @metalmatze @GiedriusS  PTAL

### CODE REVIEW

File: docs/getting-started.md [hunk 1]
```diff
@@ -37,6 +37,12 @@ See [release process docs](release-process.md) for details.
 
 For detailed, free in-browser interactive tutorial please visit our [Katacoda Thanos Course](https://katacoda.com/bwplotka/courses/thanos)
 
+### Kubernetes
+If you want to run Thanos in Kubernetes, please refer to the 
+[kube-thanos](https://github.com/thanos-io/kube-thanos) project providing jsonnet library to generate kubernetes manifests
+or take a look at the already [pre-generated manifests](https://github.com/thanos-io/kube-thanos/tree/master/manifests).
```
Comment (bwplotka) [inline docs/getting-started.md]: Maybe 
```suggestion
   * take a look at the already [pre-generated manifests](https://github.com/thanos-io/kube-thanos/tree/master/manifests).
```
It's not `or` it's kind of part of kube-thanos (:

**Comment by bwplotka** [comment]
cc @FUSAKLA rebase

**Comment by FUSAKLA** [comment]
@bwplotka I saw that you covered the kubernetes references in your update so I just removed mine and left the release process update

**Comment by bwplotka** [comment]
Hi @FUSAKLA do you mind fixing links now? ): Let's  merge this finally!

**Comment by FUSAKLA** [comment]
Hi, sorry for the delay :beach_umbrella:  :))

I'll update it right away

**Comment by bwplotka** [comment]
ping @FUSAKLA  (:

File: docs/release-process.md [hunk 2]
```diff
@@ -62,6 +62,12 @@ Process of releasing a *minor* Thanos version:
     1. *In case of version after `v1+.y.z`*, double check if none of the changes break API compatibility. This should be done in PR review process, but double check is good to have.
     1. In case of `v0.y.z`, document all incompatibilities in changelog.
 
+1. Update tutorials:
+    1. Update the Thanos version used in the [tutorials](../tutorials) manifests.
+    1. In case of any breaking changes or necessary updates adjust the manifests
+       so the tutorial stays up to date.
+    1. Same way update the [scripts/quickstart.sh](../scripts/quickstart.sh) script if needed.
```
Comment (squat) [inline docs/release-process.md]: Nit: wording
```suggestion
    1. Update the [scripts/quickstart.sh](../scripts/quickstart.sh) script in the same way if needed.
```

**Comment by FUSAKLA** [comment]
Sorry for the delay, hopefully should be ok now :crossed_fingers:

**Pull request merged**